server/chat-plugins/repeats.ts

```diff
@@ -155,15 +158,16 @@ export const commands: ChatCommands = {
 			throw new Chat.ErrorMessage(this.tr`The phrase labeled with "${id}" is already being repeated in this room.`);
 		}
 
-		if (isHTML) this.checkHTML(message);
+		if (isHTML) this.checkHTML(phrase);
 
 		Repeats.addRepeat(room, {
 			id,
-			phrase: isHTML ? message : Chat.formatText(message).replace(/\n/g, `<br />`),
+			phrase,
 			interval: interval * 60 * 1000, // convert to milliseconds
+			isHTML,
 		});
 
-		this.modlog('REPEATPHRASE', null, `every ${interval} minute${Chat.plural(interval)}: "${message}"`);
+		this.modlog('REPEATPHRASE', null, `every ${interval} minute${Chat.plural(interval)}: "${phrase}"`);
```

Comment:
```suggestion
		this.modlog('REPEATPHRASE', null, `every ${interval} minute${Chat.plural(interval)}: "${phrase.replace(/\n/g, ' ')}"`);
```
(Modlog entries containing newlines won't be supported until the SQLite migration is complete.)